dashboard/src/main/resources/templates/dashboard.ftl

```diff
@@ -49,18 +49,39 @@
         </tr>
       </#list>
     </table>
+    
+    <hr />
+      
+    <h2>Recommended Versions</h2>
       
-     <h2>Recommended Versions</h2>
+    <p>These are the most recent versions of dependencies used by any of the covered artifacts.</p> 
       
-     <p>These are the most recent versions of dependencies used by any of the covered artifacts.</p> 
+    <ul>
+      <#list latestArtifacts as artifact, version>
+        <li>${artifact}:${version}</li>
+      </#list>
+    </ul>
+ 
+    <hr />
       
-     <ul>
-       <#list latestArtifacts as artifact, version>
-         <li>${artifact}:${version}</li>
-       </#list>
-     </ul>
+    <h2>Pre 1.0 Versions</h2>
       
-      <hr />
-      <p id='updated'>Last generated at ${lastUpdated}</p>
+    <p>
+      These are dependencies found in the GCP orbit which have not yet reached 1.0.
+      No 1.0 or later library should depend on them.
+      If the libraries are stable, advance them to 1.0.
+      Otherwise replace the dependency with something else.
+    </p> 
+    
+    <ul id="unstable">
+      <#list latestArtifacts as artifact, version>
+        <#if version[0] == '0'>
+          <li>${artifact}:${version}</li>
+        </#if>
+      </#list>
+    </ul>
```

Comment:
It appears that this will just be empty if there are no 0.x libraries; do we want to print a "yay, you have no unstable deps" message in that case?

Reply:
Ideally, though this doesn't seem likely to happen for some time, and putting logic like this in FreeMarker is way more convoluted than it should be.

Reply:
If we don't resolve it though, it's one of those things that will pop up later and it will confuse us until we remember what happened.

Reply:
done